action.yaml

```diff
@@ -94,6 +94,9 @@ inputs:
   flakehub-push-url:
     description: A URL pointing to a `flakehub-push` binary. Overrides all other `flakehub-push-*` options.
     required: false
+  include-output-paths:
```

Comment:
Should should be moved up below `error-on-conflict`, these knobs are more dev-only.